CONTRIBUTING.md

```diff
@@ -83,8 +83,9 @@ You won't be yelled at for giving your best effort. The worst that can happen is
 
 1. Follow the link above or follow instructions to [create a new Codespace.](https://docs.github.com/en/codespaces/developing-in-a-codespace/creating-a-codespace-for-a-repository); You can use the web editor, or even better open the Codespace in VSCode
 2. Wait for the container to start. This will take a few (10-15) minutes since Ruby needs to be installed, the database needs to be created, and the `bin/setup` script needs to run
-3. Run `bin/start` and visit the URL that pops in VSCode up to see the human essentials page
-4. Login as a sample user with the default [credentials](#credentials).
+3. Run `bin/start`.
+4. Right click on port 3000, set the port visibility to Public, and visit the URL that pops in VSCode up to see the human essentials page. 
```

Comment:
I tried again today and now the private port is working so I think it may have been a bug in the VSCode Codespaces extension that got fixed with yesterday's release. Yesterday I tried to view the app at 127.0.0.1:3000 after port forwarding with visibility set to private and couldn't figure out why it wasn't working, I just got "ERR_EMPTY_RESPONSE". The server was up and running and I could curl to it just fine in the Codespace terminal, so I really couldn't figure out what was wrong. There was another contributor in Slack who ran into this issue also.